cli/processexecutor.cpp

```diff
@@ -331,11 +333,18 @@ unsigned int ProcessExecutor::check()
                             std::size_t size = 0;
                             if (p != pipeFile.cend()) {
                                 pipeFile.erase(p);
-                                const auto fs = std::find_if(mFiles.cbegin(), mFiles.cend(), [&name](const FileWithDetails& entry) {
-                                    return entry.path() == name;
+                                const auto fs = std::find_if(mFileSettings.cbegin(), mFileSettings.cend(), [&name](const FileSettings& entry) {
+                                    return entry.filename() == name.substr(0, name.find(' '));
```

Comment:
hmm.. this does not work well when there is a space in the path right?

A typical scenario when there is a space in the path would be a windows user that has source code in his home folder i.e. "C:\Users\Fred Flintstone\helloworld\..."

Reply:
hmm.. ideally we would add a test where the filenames have spaces but hmm then you can't reuse the existing testfolder. so I suggest you put the test in "test_project.py" instead?
I assume it's enough that the filename contains spaces in the test the path doesn't have to have spaces.

Reply:
Do you mean project_test.py?

Reply:
I do not understand what this code is actually doing?

Reply:
As I understand it the `pipeFile` maps pipe fds to filenames, but for `FileSettings` it also tacks on the configuration. So this is finding the original entry. But i guess it would me better to just store the filesize in `pipeFile` as well...

Reply:
I see. We could have just used a pointer/reference to the `FileWithDetails` object. But it doesn't matter as that code will go away at some point anyways.